subsys/bluetooth/services/bas_c.c

```diff
@@ -13,6 +13,7 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(bas_c, CONFIG_BT_GATT_BAS_C_LOG_LEVEL);
 
+#define BAS_PERIODIC_READ_PROCESS 0
```

Comment:
```BAS_PERIODIC_READ_PROC_BIT``` - anything that would explain that this is bit position. I had to search it in the code to find out that it is not a kind of a setting.

Reply:
Right. Changed